src/com/lftechnology/java_training/dec31/DistanceCalculate.java

```diff
@@ -0,0 +1,158 @@
+package com.lftechnology.java_training.dec31;
+
+import java.util.Scanner;
+
+/**
+ * a program that contains a class called Distance that includes two private
+ * data members feet (type int) and inches (type float). Use a method setData to
+ * set values to instance variables. Provide a method display that displays the
+ * feet and inches, a member function addDistance for adding two distances, and
+ * a member function compareDistance for comparing two distances.
+ * 
+ * 
+ * 
+ */
+class Distance {
+	private int feet = 0;
+	private float inches = 0;;
+	private Scanner input = new Scanner(System.in);
+
+	/**
+	 * set values to instance of variable
+	 */
+	protected void setData(int feet, float inches) {
```

Comment:
Protected is used if the class have their child classes i.e if the method should made accessible to those class which extends Distance class. Please re-think